modules/ROOT/pages/platform/security.adoc

```diff
@@ -6,13 +6,27 @@
 label:AuraDB-Enterprise[]
 label:AuraDS-Enterprise[]
 
-AuraDB Enterprise and AuraDS Enterprise run within a Virtual Private Cloud (VPC) isolation for your deployment.
+AuraDB Enterprise and AuraDS Enterprise run in a dedicated cloud Account (AWS), Subscription (Azure) or Project (GCP) to achieve complete isolation for your deployment.
 
-The VPC enables you to operate within an isolated section of the service, where your processing, networking, and storage are further protected.
+Additional VPC boundaries enable you to operate within an isolated section of the service, where your processing, networking, and storage are further protected.
 
-Please note that the Aura Console runs in a separate VPC.
+Note that the Aura Console runs in a separate VPC, separate from the rest of Aura.
 
-=== AWS Private endpoints
+== Network Access
+
+Neo4j provides authorized users with the ability to configure certain aspects of the infrastructure that is used to run and manage Aura instances, and the networking the platform leverages to establish secure connections between the database (Aura) and application VPCs.
+This includes the ability to connect over the cloud provider's private link and private endpoint.
+
+You can also choose whether traffic to your Aura instance is allowed to traverse the public internet, or if routes should be restricted to the private link by disabling public traffic. 
+To enable this, create a private endpoint in your own VPC. 
```

Comment:
```suggestion
You can also choose whether traffic to your Aura instance is allowed to traverse the public internet, or if routes should be restricted to the private link by disabling public traffic. 
To enable this, create a private endpoint in your own VPC. 
```
To enable what? Traverse the public internet or restrict traffic?

Reply:
In order to control traffic and restrictions, customers will need to create a private endpoint in their own VPC.